terraform/aws/pipeline/default/postgres/main.tf

```diff
@@ -0,0 +1,124 @@
+module "common" {
```

Comment:
Following this pattern forward with Snowflake or other loaders doesn't this break the ability to have multiple connections as each "common" module is going to deploy new collector + enrich + s3 loaders?

Seems like it would make sense to just keep them segmented and instruct a user to first deploy the common part and that they can then deploy postgres / snowflake / etc.

Reply:
Yep, makes sense. I will make necessary changes 👍